is_real_robot: true
device:
  name: "hopper_teststand"
  sensors:
    # joints data
    joint_positions:
      size: 1
    joint_velocities:
      size: 1
    joint_torques:
      size: 1
    joint_target_torques:
      size: 1
    joint_encoder_index:
      size: 1
    # additional data
    slider_positions:
      size: 1
  controls:
    ctrl_joint_torques:
      size: 1
hardware_communication:
  max_missed_control: 10
  control_period: 1000000 # in nano second this means 1ms <=> 1kz

debug_timer_history_length: 0

motor_KT: 0.025    # The Kt constant of the motor [Nm/A]: tau = I * Kt
